conjurapi/authn/oidc_authenticator.go

```diff
@@ -0,0 +1,31 @@
+package authn
+
+import (
+	"net/http"
+	"net/url"
+
+	"github.com/cyberark/conjur-api-go/conjurapi/response"
+)
+
+type OIDCAuthenticator struct {
+	AuthenticateURL string `env:"CONJUR_AUTHN_OIDC_URL"`
```

Comment:
The url can be composed of host + `/authn-oidc/` + authenticator name so we should maybe try to figure out how to compose this from other variables but that can be future work.